enlace/programs/profile.php

```diff
@@ -297,14 +302,17 @@ function(response) {
                 -->
                 <table class="inner_table">
                     <tr><th>Session</th><th>Number Surveys Completed / Number Participants in the Session</th><th>Percent Surveys Completed</th></tr>
+                    <tr><td colspan="4"><h5 onclick="$('.oldsession_surveys').toggle();" style="cursor:pointer;">Old Sessions<span class="oldsession_surveys"> (Hidden)</span></h5></td></tr>
                     <?php
                     //get each session
                     $get_sessions = "SELECT * FROM Session_Names WHERE Program_ID='$program->program_id'";
                     include "../include/dbconnopen.php";
                     $sessions = mysqli_query($cnnEnlace, $get_sessions);
-                    while ($sesh = mysqli_fetch_row($sessions)) {
+                    while ($sesh = mysqli_fetch_array($sessions)) {
+                        $old = (strtotime($sesh['End_Date']) < strtotime("-$num_days_hidden days"));
```

Comment:
This line implements the behavior whose prose description I didn't understand earlier?